learn-ocaml.el

```diff
@@ -57,3 +57,28 @@
 		 :fetch t)
        :stderr learn-ocaml-log-buffer
        :buffer learn-ocaml-log-buffer ))
+
+(defun learn-ocaml-file-writter-filter (proc string)
+  (write-region string nil learn-ocaml-temp t)
+  )
+
+(cl-defun learn-ocaml-grade-file (&key id token server dont-submit file)
+  "enables the user to upload a file to the server"
```

Comment:
Suggestion of docstring:

"Grade a .ml file, optionally submitting the code and the note to the server."